app/buck2_common/src/http/retries.rs

```diff
@@ -64,11 +75,18 @@ where
         if let Some(http_error) = http_error {
             if http_error.is_retryable() {
                 if let Some(b) = backoff.peek() {
-                    tracing::warn!(
-                        "Retrying a HTTP error after {} seconds: {:#}",
-                        b.as_secs(),
-                        http_error
-                    );
+                    let secs = b.as_secs();
```

Comment:
Could we maybe check elapsed + this backoff, and then use that and the 30 second threshold? I'm a bit concerned about not showing anything until we start facing a very large backoff.

Reply:
I also note that I can' find any places that retry for 30 seconds so this really just amounts to just deleting the log message.